code/modules/mob/living/carbon/xenomorph/castes/hivemind/hivemind.dm

```diff
@@ -133,17 +134,34 @@
 	icon = 'icons/Xeno/weeds.dmi'
 	icon_state = "weed_hivemind4"
 	var/mob/living/carbon/xenomorph/hivemind/parent
+	var/core_regeneration = 25
 
 /obj/effect/alien/hivemindcore/Initialize(mapload)
 	. = ..()
 	set_light(7, 5, LIGHT_COLOR_PURPLE)
+	START_PROCESSING(SSprocessing, src)
+
+/obj/effect/alien/hivemindcore/examine(mob/user)
+	. = ..()
+	if(!isxeno(user))
+		return
+	var/list/details = list()
+	details +=("This [name] belongs to [parent.name].</br>")
+	details +=("It has [obj_integrity] of [max_integrity] integrity remaining.</br>")
+
```

Comment:
Just have this split with an \n no need for the list

Reply:
Done.